tests/test_transformers.py

```diff
@@ -767,30 +767,10 @@ def test_binarize_works_in_gridsearch(self, train_iris_dataset):
 
 
 class TestRareFeatureEncoder(TransformerBase):
-    def test_rare_feature_encoder_returns_correctly_series(self):
-        threshold = 2
-        fill_rare = "Rare"
-        rare = RareFeatureEncoder(threshold=threshold, fill_rare=fill_rare)
-
-        data = pd.DataFrame({"categorical": [1, "a", "a", 2, "b", 1]})
-        rare.fit(data)
-
-        new_data = pd.DataFrame({"categorical": [1, 1, 1, "a", "b", "b", 3]})
-        result = rare.transform(new_data)
-
-        test_data = pd.DataFrame({"categorical": [1, 1, 1, "a", "Rare", "Rare", 3]})
-
-        assert isinstance(result, pd.DataFrame)
-        assert len(new_data) == len(result)
-        assert test_data.equals(result)
-
-    def test_rare_feature_encoder_returns_correctly_dataframe(self):
-        threshold = 2
-        fill_rare = 99
-        rare = RareFeatureEncoder(threshold=threshold, fill_rare=fill_rare)
-
-        data = pd.DataFrame({"categorical": [1, "a", "a", 2, "b", 2]})
-        rare.fit(data)
+    def test_rare_feature_encoder_returns_correctly_when_given_one_column_dataframe(
```

Comment:
"Correctly" is a bit vague here.

Now it is testing if the transformed data and the input data have the same shape - maybe we can specify that in the function name?